automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/Modals.scala

```diff
@@ -63,5 +63,11 @@ case class MessageModal(implicit webDriver: WebDriver) extends OKCancelModal {
   def validateLocation: Boolean = {
     testId("message-modal-content").element != null
   }
+
```

Comment:
fyi @ahaessly is adding the same functionality
https://github.com/broadinstitute/firecloud-ui/pull/1109/files#diff-7a49eedc1ad01e518a6547aec80ab6f9R67